schemas/doi.json

```diff
@@ -38,6 +38,11 @@
 	  "validation": [
 		"in:1,2,3,4,5"
 	  ]
+	},
+	"registrationType": {
```

Comment:
What about existing DOIs? Do they get a `registrationType` set during migration?

Reply:
I was wondering about this. This detail is known at the time of migration, but is effectively removed as part of the initial DOI migration process. Seeing as we haven't hit a release yet, would you have any objections to updating this migration step (`I7014_DoiMigration`)? Would also be able to differentiate between marked registered and registered via the plugin for existing DOIs at this stage, too.

Reply:
Yeah it's perfectly fine to update a migration before it's released.